ui/client/components/graph/node-modal/editors/expression/ExpressionField.tsx

```diff
@@ -39,7 +39,7 @@ class ExpressionField extends React.Component<Props> {
 
     if (editor.type === EditorType.FIXED_VALUES_PARAMETER_EDITOR)
       return (
-        <EditableExpression
+        <EditableEditor
```

Comment:
Maybe you can extract creating `EditableEditor` to another method?

Reply:
It would be not related to this change - the only thing which changed in this file is because of renaming :)

Reply:
Oka ;)